products/BellHybrid/apps/common/src/UserSessionModel.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "models/UserSessionModel.hpp"
+#include "log/log.hpp" //TODO: is it still needed?
```

Comment:
It is, as you're logging from inside this file :wink: But please include it with angle brackets